contract/src/lib.rs

```diff
@@ -70,10 +68,29 @@ pub const NFT_STANDARD_NAME: &str = "nep171";
 #[near_bindgen]
 #[derive(BorshDeserialize, BorshSerialize, PanicOnDefault)]
 pub struct Contract {
+    //status of initiation
+    pub is_init: bool,
+
     //contract owner
     pub owner_id: AccountId,
 
-    pub is_init: bool,
+    //keeps track of the id_attempts vec for a given account ID
+    pub id_attempts: UnorderedMap<AccountId, Vec<String>>,
+
+    //keeps track of the tickets vec for a given individual key
+    subjects: UnorderedMap<String, Vec<Section>>,
+
+    //keeps track of the answers vec for a given individual key
+    pub answers: UnorderedMap<String, Vec<Answer>>,
+
+    //keeps track of the answers_id for a given attempt_key
+    pub key_answers_id: LookupMap<String, UnorderedSet<String>>,
```

Comment:
There's no point in using near collections as values to other near collections. Because these collections are optimized to access blockchain data, and you've already accessed the data by using `LookupMap`. So, my suggestion is to use some simple Rust set like `HashSet` here.